webview.h

```diff
@@ -992,6 +999,15 @@ class edge_chromium : public browser {
     m_webview->ExecuteScript(wjs.c_str(), nullptr);
   }
 
+  std::string get_url() {
```

Comment:
This method uses deprecated string conversion functions and you will receive compiler warnings. Use `winrt::to_string()`

Reply:
Thank you for letting me know. My compiler must be old, so I will update it.